packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -279,6 +280,8 @@ class DriveCommand extends RunCommandBase {
         packageConfig,
         chromeBinary: stringArgDeprecated('chrome-binary'),
         headless: boolArgDeprecated('headless'),
+        webBrowserFlags: stringArgDeprecated(FlutterOptions.kWebBrowserFlags)
+            ?.split(RegExp(r'\s+')),
```

Comment:
What if there are spaces within an option, such as `--auto-select-desktop-capture-source="Entire screen"`?

Reply:
hmm yeah, could you make this a multi arg instead of splitting yourself?

like --web-browser-flag="--foo" --web-browser-flag="--bar=fizz"

That should work...

Reply:
Nice @christopherfujino @jonahwilliams! I now adapted the option accordingly. I also thought of a comma separated approach, but this has the same disadvantages as the space separation (e.g. would not cover `--my-dimension-flag=512,1024`